### PR TITLE
translation for agentcore-core

### PR DESCRIPTION
**Pull request by lilai23**
**What type of PR is this?**

Doc

**What this PR does / why we need it?**

This initiative aims to enhance readability, accessibility, and collaboration for both current and potential contributors, ensuring a unified language throughout the codebase.

**Which issue(s) this PR fixes？**

Fixes #1466

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [ ] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java [hunk 1]
```diff
@@ -20,105 +20,105 @@
...
 
     /**
-     * pluginPackage插件包的键
+     * key of pluginPackage
      */
     public static final String PLUGIN_PACKAGE_DIR_KEY = "plugin.package.dir";
 
     /**
-     * logback配置的键
+     * key of log setting file
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: key of logback setting file

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/AgentCoreEntrance.java [hunk 2]
```diff
@@ -53,31 +53,31 @@ public class AgentCoreEntrance {
     private static final Logger LOGGER = LoggerFactory.getLogger();
 
     /**
-     * 缓存当前Agent的类型，默认为premain方式启动
+     * Caches the type of the current Agent. The default mode is premain
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/AgentCoreEntrance.java]: The default startup mode is premain

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/utils/ConfigFieldUtil.java [hunk 3]
```diff
@@ -27,48 +27,49 @@
...
 
     /**
-     * boolean类型的get或set方法中属性名的最短长度
+     * TFIELD_NAME_MIN_LENGTH
      */
     private static final int FIELD_NAME_MIN_LENGTH = 3;
 
     /**
-     * boolean类型的get或set方法中属性名的首字母下标
+     * FIELD_NAME_CHECK_INDEX
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/utils/ConfigFieldUtil.java]: Initial letter subscript of the property name in the get or set method of the boolean type.

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java [hunk 4]
```diff
@@ -20,105 +20,105 @@
...
 
     /**
-     * 核心包版本键
+     * sermant version
      */
     public static final String CORE_VERSION_KEY = "Sermant-Version";
 
     /**
-     * 启动配置中应用名称的键
+     * appName
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: The key of the application name

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java [hunk 5]
```diff
@@ -20,105 +20,105 @@
...
      */
     public static final String APP_NAME_KEY = "appName";
 
     /**
-     * 启动配置中应用类型的键
+     * appType
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: The key of the application type

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java [hunk 6]
```diff
@@ -20,105 +20,105 @@
...
      */
     public static final String APP_TYPE_KEY = "appType";
 
     /**
-     * 启动配置中服务名称的键
+     * serviceName
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: The key of the service name

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/ConfigManager.java [hunk 7]
```diff
@@ -200,12 +209,14 @@ private static LoadConfigStrategy<?> getLoadConfigStrategy(File configFile, Clas
...
 
     /**
-     * 遍历所有通过spi方式声明的配置对象
-     * <p>需要在{@code META-INF/services}目录中添加配置基类{@link BaseConfig}文件，并添加实现
-     * <p>文件中声明的所有实现都将会进行遍历，每一个实现类都会通过spi获取实例，然后调用{@code configConsumer}进行消费
+     * Iterate over all configuration objects declared by spi
+     * <p>need to add the configuration base class {@link BaseConfig} file in the {@code META-INF/services} directory
+     * and add the implementation
+     * <p>All implementations declared will be iterated, and each implementation class will get an
+     * instance through spi, and then consume byconfigConsumer
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/ConfigManager.java]: then call {@code configConsumer} to consume

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/Plugin.java [hunk 8]
```diff
@@ -28,74 +28,75 @@
 import java.util.Set;
 
 /**
- * 用于维护插件信息 插件名，插件包目录，插件服务，插件配置，插件主模块的类加载器，插件服务类加载器，插件的ResetTransformer
+ * Used to maintain plugin information plugin name, plugin package directory, plugin service, plugin configuration, main
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/Plugin.java]: Used to maintain plugin information：

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java [hunk 9]
```diff
@@ -224,18 +231,19 @@ public void onTransformation(TypeDescription typeDescription, ClassLoader classL
...
+     * @return BufferedAgentBuilder Wrapper of AgentBuilder
      */
     public BufferedAgentBuilder addPlugins(Iterable<PluginDescription> plugins) {
         return addAction(new BuilderAction() {
             @Override
             public AgentBuilder process(AgentBuilder builder) {
                 AgentBuilder newBuilder = builder;
                 for (PluginDescription plugin : plugins) {
-                    // 此处必须赋值给newBuilder，不可在原builder上重复操作，否则上次循环中的操作会不生效
+                    // This must be assigned to the new Builder, and the operation cannot be repeated on the original
```
Comment (daizhenyu) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java]: newBuilder

Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java]: This must be assigned to the newBuilder

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/service/dynamicconfig/config/DynamicConfig.java [hunk 10]
```diff
@@ -83,7 +83,7 @@ public class DynamicConfig implements BaseConfig {
     private boolean enableAuth = false;
 
     /**
-     * 请求超时时间，nacos每次getConfig需要设置一个超时时间
+     * Request timeout. Nacos needs to set one timeout per get config
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/service/dynamicconfig/config/DynamicConfig.java]: Nacos needs to set one timeout per getConfig

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/LabelGroupUtils.java [hunk 11]
```diff
@@ -156,10 +157,10 @@ public static Map<String, String> resolveGroupLabels(String group) {
     }
 
     /**
-     * 获取标签信息
+     * get label condition
      *
-     * @param group 分组  app=sc&service=helloService转换label=app:sc&label=service:helloService
-     * @return 标签组条件
+     * @param group group, app=sc&service=helloService转换label=app:sc&label=service:helloService
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/LabelGroupUtils.java]: app=sc&service=helloService convert  label=app:sc&label=service:helloService

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/StringUtils.java [hunk 12]
```diff
@@ -165,10 +165,10 @@ public static boolean isWildcardMatch(String str, String wc) {
     }
 
     /**
-     * 获取对象的toString信息
+     * Gets the object's to String information
```
Comment (hanbingleixue) [inline sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/StringUtils.java]: Gets the object's toString information

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java (hunk 1 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/utils/ConfigFieldUtil.java (hunk 3 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/utils/ConfigFieldUtil.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/Plugin.java (hunk 8 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/Plugin.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java (hunk 9 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/AgentCoreEntrance.java (hunk 2 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/AgentCoreEntrance.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java (hunk 4 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java (hunk 5 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java (hunk 6 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/common/CommonConstant.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/ConfigManager.java (hunk 7 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/config/ConfigManager.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java (hunk 9 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/BufferedAgentBuilder.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/service/dynamicconfig/config/DynamicConfig.java (hunk 10 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/service/dynamicconfig/config/DynamicConfig.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/LabelGroupUtils.java (hunk 11 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/LabelGroupUtils.java]: done

File: sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/StringUtils.java (hunk 12 above)
Comment (lilai23) [reply sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/utils/StringUtils.java]: done

**Review by daizhenyu — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**